expressEngine/server.js

```diff
@@ -28,6 +29,7 @@ app.use('/health-check', (req, res, next) => {
 app.use('/api-docs', SwaggerUi.serve, SwaggerUi.setup(swaggerDocument));
 
 app.use('/api', Route);
+app.use(errorHandler);
```

Comment:
Can you please explain to me why you are using the errorHandler file in server.js

Reply:
I am using error handler as a middleware, if any error comes in the other routes it will be handled by error handler middleware so i have written it after other routes and used in server.js.

Reply:
If you are using error handler as middleware then you must call it in route.js

Reply:
This is because if we add other routes in server.js it can handle those routes errors also.